docs/common/craft-parts/reference/part_properties.rst

```diff
@@ -69,7 +69,10 @@ build-snaps
 
 The snaps to be installed in the build environment before the build is
 performed. These provide libraries and executables that the part needs during
-the build process.
+the build process. They take the form of ``<snap>/<version>/<channel>``,
+``<snap>/<channel>`` or just ``<snap>``. For example, the ``juju`` snap could be
+specified as ``juju/2.9/stable``, ``juju/latest/stable`` or ``juju/stable`` to
```

Comment:
Why the `juju` example? It seems like an uncommon snap to use as an example.

Reply:
I changed it to `node`.